pkg/cloud/vsphere/context/machine_context.go

```diff
@@ -230,7 +230,7 @@ func (c *MachineContext) Patch() error {
 	c.Machine.Status.ProviderStatus = newStatus
 
 	if !reflect.DeepEqual(c.Machine.Status, c.MachineCopy.Status) {
-		c.Logger.V(1).Info("updating machine status")
+		c.Logger.V(2).Info("updating machine status", "task", c.MachineStatus.TaskRef)
```

Comment:
Hi @moshloop,

I like including the task, but instead of as a field, let's optionally add it as a namespace to the logger. It makes more sense since the task may not be present at all.